{"name": "test_reg_several_incorrect_fields", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Incorrect username length, incorrect email address, passwords must match' in '<html lang=\"en\"><head>\\n    <meta charset=\"UTF-8\">\\n    <meta http-equiv=\"Content-Type\" content=\"text/html; charset=u.../form>\\n                </div>\\n            </div>\\n        </div>\\n    </div>\\n    </div>\\n</div>\\n\\n\\n</body></html>'\n +  where '<html lang=\"en\"><head>\\n    <meta charset=\"UTF-8\">\\n    <meta http-equiv=\"Content-Type\" content=\"text/html; charset=u.../form>\\n                </div>\\n            </div>\\n        </div>\\n    </div>\\n    </div>\\n</div>\\n\\n\\n</body></html>' = <selenium.webdriver.remote.webdriver.WebDriver (session=\"9d69ffc34ebb4ac46b3791e992ef81a3\")>.page_source\n +    where <selenium.webdriver.remote.webdriver.WebDriver (session=\"9d69ffc34ebb4ac46b3791e992ef81a3\")> = <tests.test_ui.TestReg object at 0x119d940d0>.driver", "trace": "self = <tests.test_ui.TestReg object at 0x119d940d0>\n\n    def test_reg_several_incorrect_fields(self):\n        \"\"\"\n        Тестируется сценарий, когда пользователь при регистрации допустил ошибки в более чем 1-м поле ввода.\n        Приложение выводит ошибку в виде словаря из некорректных данных, не прилагая пояснительный текст с описанием\n        возникших проблем для обычного пользователя\n        \"\"\"\n        username, email, password = generate_user(len_user=1, mask=False)\n        self.reg_page.reg(username, email, password, generate_user()[2])\n        allure.attach(self.driver.get_screenshot_as_png(),\n                      name='test_python_link',\n                      attachment_type=allure.attachment_type.PNG)\n>       assert \"Incorrect username length, incorrect email address, passwords must match\" in self.driver.page_source\nE       assert 'Incorrect username length, incorrect email address, passwords must match' in '<html lang=\"en\"><head>\\n    <meta charset=\"UTF-8\">\\n    <meta http-equiv=\"Content-Type\" content=\"text/html; charset=u.../form>\\n                </div>\\n            </div>\\n        </div>\\n    </div>\\n    </div>\\n</div>\\n\\n\\n</body></html>'\nE        +  where '<html lang=\"en\"><head>\\n    <meta charset=\"UTF-8\">\\n    <meta http-equiv=\"Content-Type\" content=\"text/html; charset=u.../form>\\n                </div>\\n            </div>\\n        </div>\\n    </div>\\n    </div>\\n</div>\\n\\n\\n</body></html>' = <selenium.webdriver.remote.webdriver.WebDriver (session=\"9d69ffc34ebb4ac46b3791e992ef81a3\")>.page_source\nE        +    where <selenium.webdriver.remote.webdriver.WebDriver (session=\"9d69ffc34ebb4ac46b3791e992ef81a3\")> = <tests.test_ui.TestReg object at 0x119d940d0>.driver\n\ntests/test_ui.py:153: AssertionError"}, "description": "\n        Тестируется сценарий, когда пользователь при регистрации допустил ошибки в более чем 1-м поле ввода.\n        Приложение выводит ошибку в виде словаря из некорректных данных, не прилагая пояснительный текст с описанием\n        возникших проблем для обычного пользователя\n        ", "attachments": [{"name": "test_python_link", "source": "cd7ac958-f7e7-46b1-82a7-66c79aae16b9-attachment.png", "type": "image/png"}], "start": 1590610545633, "stop": 1590610555218, "uuid": "b9f72596-ced4-41e5-bb91-f19836f97951", "historyId": "ec37ae3291d62a45e3384fb1b16259db", "testCaseId": "f23457473e23b7c0f22511935219ca77", "fullName": "tests.test_ui.TestReg#test_reg_several_incorrect_fields", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_ui"}, {"name": "subSuite", "value": "TestReg"}, {"name": "host", "value": "MacBook-Air-Roman-2.local"}, {"name": "thread", "value": "20863-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_ui"}]}